src/installer/bin.ts

```diff
@@ -103,6 +103,7 @@ function run(): void {
     console.log(chalk.red(err.message.trim()))
     debug(err.stack)
     console.log(chalk.red(`husky > Failed to ${action}`))
+    process.exit(1)
```

Comment:
Better use `process.exitCode`, since `process.exit` might prevent the logs from being flushed:
```suggestion
    process.exitCode = 1;
```

Reply:
Sure 👍 
Remove the semicolon and i'll commit your suggestion.

This repo has semi set to false https://github.com/typicode/husky/blob/0ffe7853853dd7667ddc24a78469b92b5312aeb2/.prettierrc#L2

Reply:
Sure 👍 
Remove the semicolon and i'll commit your suggestion.

This repo has semi set to false https://github.com/typicode/husky/blob/0ffe7853853dd7667ddc24a78469b92b5312aeb2/.prettierrc#L2

Reply:
Woops, updated 👍